packages/vapor/scss/components/tooltip.scss

```diff
@@ -69,28 +69,6 @@
         }
     }
 
-    &.state-error {
```

Comment:
![image](https://user-images.githubusercontent.com/52677246/105499995-42299700-5c90-11eb-9041-06f452c10a60.png)
I find this class in these 2 files, is there any breaking change?

Reply:
hmm I don't know, I can't find a way to reproduce the case where these classes are used. I'll try again later today